lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/LocalClient.java

```diff
@@ -6,9 +6,9 @@
  * to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p/>
```

Comment:
Please undo the change to the license header. See [REEF-315](https://issues.apache.org/jira/browse/REEF-315)